tests/integration/integration.js

```diff
@@ -75,20 +75,20 @@ define([
 				if (lodash.isFunction(v)) { return v(other[k]); }
 				else { return lodash.isEqual(v, other[k]); }
 			});
-			return lodash.all(result);
+			return lodash.every(result);
 		}
 
 		function strip(logLine) {
 			var expectedKeys = lodash.keys(expected);
-			var stripped = lodash.pick(logLine, function (v, k) { return lodash.include(expectedKeys, k); });
+			var stripped = lodash.pickBy(logLine, function (v, k) { return lodash.includes(expectedKeys, k); });
```

Comment:
Want to import these lodash methods separately as you did in other files?

Reply:
Ah, it's a test file, it might be fine like this

Reply:
It works this way for now, but I'll add them properly when I upgrade to intern v4 since that changes things a lot.